pyfar/dsp/filter.py

```diff
@@ -976,3 +976,189 @@ def _coefficients_fractional_octave_bands(
                 order, Wn, btype=btype, output='sos')
         sos[idx, :, :] = sos_coeff
     return sos
+
+
+def reconstructing_fractional_octave_bands(
+        signal, num_fractions=1, frequency_range=(63, 16000),
+        overlap=1, slope=0, n_samples=2**12, sampling_rate=None):
+    """
+    Perfectly reconstructing fractional octave filter bank.
```

Comment:
1. I wouldn't call this perfectly, since the reconstruction error potentially depends on the length of the FIR filter. Even the paper calls it Orthogonal-like and not orthogonal. Maybe there's a better name for this.
2. Should we add some information on how this is different from the IIR filter bank?

Reply:
Yes, it was merely the "perfectly" I disagreed with :)